ProcessMaker/modeler
# Add message flows

Fixes #220.

**How to test:**
* Ensure you can create message flows according to the rules described in the ticket linked above: https://github.com/ProcessMaker/modeler/issues/220#issuecomment-462840889.

**Important changes:**
* New tests added for message flows; ensure all tests pass (run `npm test`).
* A message flow button has been added to the element crown.
